.github/workflows/coding-standards.yaml

```diff
@@ -0,0 +1,26 @@
+name: Check Code standards
+on: [push]
```

Comment:
Maybe we should rename this file to `push.yaml`, say, (rather than `coding-standards.yaml`) as we may add more checks to it in the future, e.g. code analysis or other fancy stuff.